src/Microsoft.VisualStudio.Services.Agent/Constants.cs

```diff
@@ -289,6 +289,8 @@ public static class Variables
             public static readonly string MacroPrefix = "$(";
             public static readonly string MacroSuffix = ")";
 
+            public static readonly string ImageVersion = "ImageVersion";
```

Comment:
Since this isn't exactly in the same bucket as the other agent and build variables (it's defined in our Azure Pipelines agent images), is there a better place to put the constant?

Reply:
Why not just under Constants and name it ImageVersionVariable? Sort of like how PathVariable is defined.

Reply:
Makes sense to me - maybe also add an explanatory comment as to why its different?